source/class/qx/tool/compiler/Analyser.js

```diff
@@ -123,6 +123,12 @@ qx.Class.define("qx.tool.compiler.Analyser", {
       nullable: false,
       check: "Array"
     },
+    
+    /** Whether and how to mangle private identifiers */
+    manglePrivates: {
+      init: "readble",
```

Comment:
Here is a typo...

Reply:
oh yes - and also a bug that the framework allows it!